docs/spark-procedures.md

```diff
@@ -587,3 +587,75 @@ Get all the snapshot ancestors by a particular snapshot
+| `changelog_view`        |   | string | Name of the view to create                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `options`     |   | map<string, string> | A map of Spark read options to use. For example, `start-snapshot-id`, the snapshot id to start reading from exclusively. If not provided, the table’s first snapshot will be used as the starting point. `end-snapshot-id`, the snapshot id to stop reading at inclusively. Default to the current snapshot. `start-timestamp`, the timestamp to start reading from exclusively. If not provided, the table’s first snapshot will be used as the starting point.`end-timestamp`, the timestamp to stop reading at inclusively. If not provided, the table’s current snapshot will be used as the ending point. | 
+|`compute_updates`| | boolean | Whether to compute updates. Defaults to false                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 
+|`identifier_columns`| | array<string> | The list of identifier columns. If not provided, and `compute_updates` is true, the table’s current identifier fields will be used.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Description should contain details on how these columns are used. In this case, 
"Used when compute_updates is true to identify pre and post update rows. If a delete and insert record have identical identifier columns, they are considered to be pre and post updates of the same row."

Or something like that